study/cyber-tutor/quiz/js/app.js

```diff
@@ -0,0 +1,97 @@
+const myQuestions = [
+  {
+    question: "What does cybersecurity protect?",
+    answers: {
+      a: "Information systems, programs, and data ",
+      b: " Houses and neighborhoods ",
+      c: " Jobs"
+    },
+    correctAnswer: "a"
+  },
+  {
+    question: "What does a hacker use to gain access to a system?",
+    answers: {
+      a: "Paper and pencil ",
+      b: " Computer ",
+      c: " Hammer"
+    },
+    correctAnswer: "b"
+  },
+];
+function createQuiz(){
+  // variable to store the HTML output
+  const output = [];
+
+  // for each question...
+  myQuestions.forEach(
+    (currentQuestion, questionNumber) => {
+
+      // variable to store the list of possible answers
+      const answers = [];
+
+      // and for each available answer...
+      for(letter in currentQuestion.answers){
+
+        // ...add an HTML radio button
+		 answers.push(
+          `<label>
+            <input type="radio" name="question${questionNumber}" value="${letter}">
+            ${currentQuestion.answers[letter]}
+          </label>`
+        );
+      }
+
+      // add this question and its answers to the output
+      output.push(
+        `<div class="question"> ${currentQuestion.question} </div>
+        <div class="answers"> ${answers.join('\n ')} </div>`
+      );
+    }
+  );
+
+  // finally combine our output list into one string of HTML and put it on the page
+  quizContainer.innerHTML = output.join('');
+}
+function showAnswers(){
+
+  // gather answer containers from our quiz
+  const answerContainers = quizContainer.querySelectorAll('.answers');
+
+  // keep track of user's answers
+  let numCorrect = 0;
+
+  // for each question...
+  myQuestions.forEach( (currentQuestion, questionNumber) => {
+
+    // find selected answer
+    const answerContainer = answerContainers[questionNumber];
+    const selector = `input[name=question${questionNumber}]:checked`;
+    const userAnswer = (answerContainer.querySelector(selector) || {}).value;
+
+    // if answer is correct
+    if(userAnswer === currentQuestion.correctAnswer){
```

Comment:
Add a space between parentheses and brackets to make things more readable.

Ex:
```js
if (userAnswer === currentQuestion.correctAnswer) {
    // Blah
}
```